### PR TITLE
docs: Added more details to docs examples

### PR DESCRIPTION
**Pull request by DenverCoder1**

### CODE REVIEW

**Review by Skelmis — APPROVED** [review]
Quickly skimmed it, lgtm

File: docs/ext/menus/button_menus.rst [hunk 1]
```diff
@@ -65,19 +65,23 @@ Instantiation is the same as for :ref:`Reaction Menus <ext_menus_reaction_menus>
...
+
+``MySource`` is the same as :ref:`defined earlier <MySource>`, but the menu is instantiated with
+:class:`ButtonMenuPages` as follows:
 
 .. code:: py
 
-    pages = menus.ButtonMenuPages(source=MySource(range(1, 100)), clear_buttons_after=True, 
-                                  style=nextcord.ButtonStyle.primary)
+    pages = menus.ButtonMenuPages(
+        source=MySource(data),
```
Comment (Skelmis) [inline docs/ext/menus/button_menus.rst:83]: Define `data`, you know people are gonna straight up copy this.

**Review by Skelmis — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -35,4 +35,4 @@ py -3 -m pip install -U nextcord-ext-menus
 ## License
 
 Copyright (c) 2021 The Nextcord Developers  
-Copyright (c) 2015-2020 Danny Y. (Rapptz)
+Copyright (c) 2020-2021 Danny Y. (Rapptz)
```
Comment (ooliver1) [inline README.md:38]: Danny's copyright extended?

Comment (DenverCoder1) [reply README.md:38]: Rapptz started development on [discord-ext-menus](https://github.com/rapptz/discord-ext-menus) in January 2020 and the last commit was from July 2021.

Comment (ooliver1) [reply README.md:38]: ah

**Review by ooliver1 — APPROVED** [review]

**Review by parker02311 — APPROVED** [review]
lgtm

**Pull request merged**